@here/harp-map-theme/package.json

```diff
@@ -10,6 +10,7 @@
     "scripts": {
         "test": "cross-env mocha --require source-map-support/register $EXTRA_MOCHA_ARGS ./test/*.js",
         "build": "tsc",
+        "prepareIcons": "ts-node ./scripts/prepareIcons.ts",
```

Comment:
We have mostly dashed-style script names all over the project.
```suggestion
        "prepare-icons": "ts-node ./scripts/prepareIcons.ts",
```

Reply:
OK, will change...